Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:3.8-slim-buster as builder
+
+ENV POETRY_VERSION=1.0.10
+
+# install poetry
+RUN pip install "poetry==$POETRY_VERSION"
+
+# set path of workdir in docker
+WORKDIR /app
+
+# copy my current dir to /app on WORKDIR
+COPY . /app
+
+# step to show my list of copied files and subdirectories
+RUN ls -la ./
+
+# turn off virtual envir creation from poetry as we're in docker
+# install poetry dependencies #COPY pyproject.toml poetry.lock ./
+RUN poetry config virtualenvs.create false \
+    && poetry install --no-interaction
+
+# Configure for production - multi build
+FROM python:3.8-slim-buster as production
+ENV POETRY_VERSION=1.0.10
+RUN pip install "poetry==$POETRY_VERSION"
+WORKDIR /app
+COPY --from=builder /app/ /app/ 
+RUN poetry config virtualenvs.create false \
+    && poetry install --no-interaction
+# set my port on container to 5001
+EXPOSE 5001
+CMD ["gunicorn", "--bind", "0.0.0.0:5001", "--log-level=debug", "app:create_app()"]
+
+# Configure for development - multi build
+FROM python:3.8-slim-buster as development
+ENV POETRY_VERSION=1.0.10
+RUN pip install "poetry==$POETRY_VERSION"
+WORKDIR /app
+COPY --from=builder /app/ /app/ 
+RUN poetry config virtualenvs.create false \
+    && poetry install --no-interaction
+EXPOSE 5002
```

Comment:
You actually mean EXPOSE 5000. The app is running on port 5000 in the container. The image doesn't care what host port you use.

Reply:
The reason I exposed port 5002 for dev (and 5001 for production) from the container is that I wanted to make sure I was actually evoking the right part of the dockerfile and the right environment. I guess in a way to prove that my localhost get cards wasn't pointing to the other environment.